db scripts/check_duplicates_ids.sql

```diff
@@ -0,0 +1,24 @@
+CREATE OR REPLACE FUNCTION public.check_duplicates_ids(curridentificationnumber character varying, investigationid integer, interactedcontactid integer)
+ RETURNS boolean
+ LANGUAGE plpgsql
+AS $function$
+declare
+	foundDuplicates boolean = false;
+	begin
+			if exists(select 1
```

Comment:
Why not using select into?

Reply:
i don't know, that's how daniel landau does it